developer-docs/installation/configuring_userprofile.md

```diff
@@ -9,6 +9,54 @@ published: true
 
 ## Overview
 
+Sunbird Release 1.11, supports user profile configuration through the system_settings table in Cassandra.
+
 ## Prerequisites
 
-## Configuring User Profile 
\ No newline at end of file
+1. Access to cassandra database
+
+## Configuring User Profile 
+
+During cassandra migration for Release 1.11 (CQL file version 1.25) will configure the following user profile configuration in system_settings table.

```

Comment:
@JaikumarSRajan Unable to understand this line. Please clarify.

Reply:
@Sunitashelar This user profile configuration is introduced in Release 1.11
It will be automatically configured in system_settings table of cassandra database when cassandra migration job for Release 1.11 is run(Running cassandra migration job is part of upgrading to any newer releases in Sunbird).
The CQL file version 1.25 holds the script to introduce this configuration during R1.11 cassandra migration